<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>378</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Exploratory analysis tools</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Chi-square test</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="open_chi_square_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>DBSCAN (vector)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="open_dbscan_vector_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>DBSCAN (raster)</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="open_dbscan_raster_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>K-means (vector)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="open_k_means_vector_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>K-means (raster)</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="open_k_means_raster_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Local Morans' I</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="open_local_morans_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Normality test (vector)</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QPushButton" name="open_normality_vector_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Normality test (raster)</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QPushButton" name="open_normality_raster_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>PCA (vector)</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QPushButton" name="open_pca_vector_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>PCA (raster)</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QPushButton" name="open_pca_raster_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>178</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
